resources/views/index.blade.php

```diff
@@ -24,6 +26,14 @@ function setCookie(name, value, expirationInDays) {
                 date.setTime(date.getTime() + (expirationInDays * 24 * 60 * 60 * 1000));
                 document.cookie = name + '=' + value + '; ' + 'expires=' + date.toUTCString() +';path=/';
             }
+            
+            function checkCookie(name) {
```

Comment:
I'd like to have all functions at the top. Move this after the body `consentWithCookies()` function.

Also rename this to `cookieExists`.